integration-test/background/request-blocking.js

```diff
@@ -106,4 +101,56 @@ describe('Test request blocking', () => {
 
         await page.close()
     })
+
+    it('serviceworkerInitiatedRequests exceptions should disable service worker blocking', async () => {
```

Comment:
If I understand it right, this test case is pretty similar to the above one, except we're turning off ServiceWorker blocking. Could we not have a function that does the work of loading the test page and getting the list of results? I figure each test case would then just need to call that function, compare the results to what's expected and optionally disable ServiceWorker initaited request blocking.